docs/book/src/roadmap.md

```diff
@@ -1,36 +1,21 @@
 # Cluster API Azure Roadmap
 
-This roadmap is a constant work in progress, subject to frequent revision. Dates are approximations. Features are listed in no particular order.
+The best place to see what's coming is in [the public milestones](https://github.com/kubernetes-sigs/cluster-api-provider-azure/milestones).  
+The next numbered milestone (i.e. **1.8**) is planned at the very beginning of the 2 month release cycle and most actively planned and discussed on the community call after the major release.
+Active community PR contributions are prioritized throughout the release and unplanned work may arise, hence the items in the milestone are a rough estimate which may change.
+The "next" milestone is where items for the milestone after the current numbered one are placed to help with a very rough bucket to help prioritize upcoming work.
 
-## v0.5 (v1alpha4) ~ Q1 2021
+## High Level Vision
 
-|Area|Description|Issue/Proposal|
-|--|--|--|
-|OS|Windows worker nodes|[#153](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/153)|
-|Identity|Multi-tenancy within one manager instance|[#586](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/586)|
-|UX|Bootstrap failure detection|[#603](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/603)|
-|UX|Add tracing and metrics|[#311](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/311)|
+CAPZ is the official production-ready Cluster API implementation to administer the entire lifecycle of self-managed or managed Kubernetes clusters (AKS) on Azure.  CAPZ enables kubernetes cluster lifecycle administration at large scale while working with GitOps and the surrounding tooling ecosystem.
```

Comment:
I like this first sentence!

I think I'd prefer something like this for the 2nd sentence:

"Cluster API defines a standard API, and provides a common toolkit, for building and maintaining Kubernetes clusters at scale in any environment."

Reply:
The 2nd sentence sounds very clean!  The concerns I have with it:
1. Cluster API and its "common toolkit" doesn't really enable scale by itself
2. "Any environment" might be a little too bold of a statement

Reply:
See what you think of the updated 2nd sentence now.